compiler/AST/symbol.cpp

```diff
@@ -2081,6 +2081,30 @@ void initAstrConsts() {
   astr_coerceMove = astr("chpl__coerceMove");
 }
 
+bool matchesOp(const char* name) {
+  if (name == astrSassign || name == astrSeq || name == astrSne ||
+      name == astrSgt || name == astrSgte || name == astrSlt ||
+      name == astrSlte || name == astrSswap || strcmp(name, "&") == 0 ||
+      strcmp(name, "|") == 0 || strcmp(name, "^") == 0 ||
```

Comment:
Seems to me these all should be astrs but you don't have to fix that in this PR

Reply:
Yeah, definitely.  I'll probably not get to that until after the release but it is something I thought about when I first added the checkParsed version